### PR TITLE
TELCODOCS-2022: Worker: pull images via cri-o

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
D/S and R/N: [MGMT-17144](https://issues.redhat.com//browse/MGMT-17144) - Worker: pull images via cri-o

Jira: https://issues.redhat.com/browse/TELCODOCS-2022

Applies to OCP version : 4.17+

Fix version: KMMO 2.2

Docs Preview: https://84978--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-example-cr_kernel-module-management-operator

Dev review: @ybettan
QE review: @ggordaniRed

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@ybettan @ggordaniRed Please review and comment/lgtm.

**Review by skrthomas — COMMENTED** [review]
Just a couple of comments, but one question I have about the ones that are `Optional: Avoid...`, I'm wondering if this should be like `Warning: Avoid...`. I realize this wording existed before, but its kinda funny to see, this is optional but avoid it.

File: modules/kmm-example-module-cr.adoc [hunk 1]
```diff
@@ -79,24 +79,25 @@ spec:
...
 +
 See "Additional resources" to learn how to replace the cluster's CA bundle.
-<11> Optional: Avoid using this parameter. If set to `true`, the build will skip any TLS server certificate validation when pulling the image in the Dockerfile `FROM` instruction using plain HTTP.
-<12> Required.
-<13> Required: A secret holding the public secureboot key with the key 'cert'.
-<14> Required: A secret holding the private secureboot key with the key 'key'.
-<15> Optional: Avoid using this parameter. If set to `true`, KMM will be allowed to check if the container image already exists using plain HTTP.
-<16> Optional: Avoid using this parameter. If set to `true`, KMM will skip any TLS server certificate validation when checking if the container image already exists.
-<17> Optional.
+<12> Optional: Avoid using this parameter. If set to `true`, the build will skip any TLS server certificate validation when pulling the image in the Dockerfile `FROM` instruction using plain HTTP.
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/kmm-example-module-cr.adoc:91]: Made this change.

File: modules/kmm-example-module-cr.adoc [hunk 2]
```diff
@@ -79,24 +79,25 @@ spec:
...
+<13> Required.
+<14> Required: A secret holding the public secureboot key with the key 'cert'.
+<15> Required: A secret holding the private secureboot key with the key 'key'.
+<16> Optional: Avoid using this parameter. If set to `true`, KMM will be allowed to check if the container image already exists using plain HTTP.
+<17> Optional: Avoid using this parameter. If set to `true`, KMM will skip any TLS server certificate validation when checking if the container image already exists.
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/kmm-example-module-cr.adoc:96]: Made these changes.

**Comment by ousleyp** [comment]
LGTM! Merging shortly.

**Pull request merged**